projects/helper/coreAssets.json

```diff
@@ -1440,7 +1440,8 @@
     "DOGS": "EQCvxJy4eG8hyHBFsZ7eePxrRsUQSFE_jpptRAYBmcG_DOGS",
     "USDe": "EQAIb6KmdfdDR7CN1GBqVJuP25iCnLKCvBlJ07Evuu2dzP5f",
     "tsUSDe": "EQDQ5UUyPHrLcQJlPAczd_fjxn8SLrlNQwolBznxCdSlfQwr",
-    "PT_tsUSDe_01Sep2025": "EQDb90Bss5FnIyq7VMmnG2UeZIzZomQsILw9Hjo1wxaF1df3"
+    "PT_tsUSDe_01Sep2025": "EQDb90Bss5FnIyq7VMmnG2UeZIzZomQsILw9Hjo1wxaF1df3",
+    "TONUSDT_STONFI": "EQCGScrZe1xbyWqWDvdI6mzP-GAcAWFv6ZXuaJOuSqemxku4"
```

Comment:
please move this address into your adapter, this doesnt meet liquidity requirements